tsconfig.json

```diff
@@ -7,12 +7,10 @@
         "target": "es2016",
         "sourceMap": false,
         "outDir": "./lib",
-        "rootDir": "./src",
         "declaration": true,
-        "typeRoots": ["src/@types"],
```

Comment:
think removing these broke the build/CI

Reply:
Thanks, will investigate

Reply:
@t3chguy sadly adding those back in (with different values to allow linting the test dir) didn't fix the CI. Any further ideas appreciated.

Reply:
I suggest comparing the output before and after your change on your dev machine to see what differs.